### PR TITLE
Improve Find Command v1

### PR DESCRIPTION
**Pull request by yockcheng**
Update find command to return names for matching substring

### CODE REVIEW

File: src/test/java/seedu/address/commons/util/StringUtilTest.java [hunk 1]
```diff
@@ -109,8 +109,11 @@ public void containsWordIgnoreCase_validInputs_correctResult() {
         assertFalse(StringUtil.containsWordIgnoreCase("    ", "123"));
 
         // Matches a partial word only
-        assertFalse(StringUtil.containsWordIgnoreCase("aaa bbb ccc", "bb")); // Sentence word bigger than query word
-        assertFalse(StringUtil.containsWordIgnoreCase("aaa bbb ccc", "bbbb")); // Query word bigger than sentence word
+
+        // Query word is substring of sentence word
+        assertTrue(StringUtil.containsWordIgnoreCase("aaa bbb ccc", "bb"));
+        // Sentence word is substring of sentence word
```
Comment (Domokunx) [inline src/test/java/seedu/address/commons/util/StringUtilTest.java:115]: The comment for this test seems incorrect?

**Review by Domokunx — CHANGES_REQUESTED** [review]
LGTM

File: src/test/java/seedu/address/commons/util/StringUtilTest.java (hunk 1 above)
Comment (yockcheng) [reply src/test/java/seedu/address/commons/util/StringUtilTest.java]: Thanks for catching it! I have made the edits accordingly

**Review by sayglenn — APPROVED** [review]
LGTM!

**Review by Domokunx — APPROVED** [review]
LGTM

**Pull request merged**